mantis-server/mantis-server-worker/build.gradle

```diff
@@ -41,6 +41,7 @@ dependencies {
     }
     compile "org.apache.httpcomponents:httpclient:$httpComponentsVersion"
     compile "io.mantisrx:mantis-rxcontrol:$mantisRxControlVersion"
+    compile "com.yahoo.datasketches:sketches-core:0.9.1"
```

Comment:
@rpalcolea what's the motivation for this addition?

Reply:
You were using this library in you code transitively, however this library came transitively:

```
com.yahoo.datasketches:sketches-core:0.9.1
+--- runtimeClasspath
\--- io.mantisrx:mantis-rxcontrol:1.3.17
     \--- runtimeClasspath (requested io.mantisrx:mantis-rxcontrol:1.3.+)
```

If you look at https://search.maven.org/artifact/io.mantisrx/mantis-rxcontrol/1.3.17/jar 

```
    <dependency>
      <groupId>com.yahoo.datasketches</groupId>
      <artifactId>sketches-core</artifactId>
      <version>0.9.1</version>
      <scope>runtime</scope>
    </dependency>
```

It comes in `runtime` scope

  In Gradle 5.x, the compile/runtime classpath separation was introduced which means that runtime dependencies won't be available in compile classpath. This is why I added it.

Reply:
Got it. Thanks for catching that.